index.html

```diff
@@ -867,7 +867,7 @@ <h2>
 
             <div class="btn-container">
               <div class="reset" data-reset="box-shadow">Reset</div>
-              <div class="btn" data-button="box-shadow">Get Results</div>
+              <div class="btn" data-button="box-shadow" id="getResultBtn">Get Results</div>
```

Comment:
This change isn't needed actually because if you check getPackage, there's already a function to get buttons